### PR TITLE
Remove UIEvent.cancelBubble

### LINKED ISSUE #7350: Remove UIEvent.cancelBubble
[It's redundant](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/cancelBubble) as there is [Event.cancelBubble](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble).

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version
   * `data:text/html,<script>alert("cancelBubble" in UIEvent.prototype)</script>` via mozregression
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: closes #7350

### CODE REVIEW

File: api/Event.json [hunk 1]
```diff
@@ -226,14 +226,24 @@
...
-              "version_added": "53",
-              "notes": "Prior to Firefox 53, this property was defined on the <a href='https://developer.mozilla.org/docs/Web/API/UIEvent'><code>UIEvent</code></a> interface. See <a href='https://bugzil.la/1298970'>bug 1298970</a> for more details."
-            },
+            "firefox": [
+              {
+                "version_added": "53"
+              },
+              {
+                "version_added": "1",
+                "notes": "This property was defined on the <a href='https://developer.mozilla.org/docs/Web/API/UIEvent'><code>UIEvent</code></a> interface."
```
Comment (saschanaz) [inline api/Event.json:235]: Could we add a dedicated field to reduce this kind of prose?

Comment (ddbeck) [reply api/Event.json:235]: Please open an issue for this. I'm not saying no, but we need to talk to consumers about whether they would actually use such a thing (or whether we need to consider something else, like generated notes).

Comment (saschanaz) [reply api/Event.json:235]: Thanks, #7400

Comment (foolip) [reply api/Event.json:235]: That was duped into https://github.com/mdn/browser-compat-data/issues/3463, good old prototype chain moves. Do we need to resolve that before finishing this PR though?

Comment (ddbeck) [reply api/Event.json:235]: Thanks for the nudge on this. I don't think we need to resolve that first. This makes some sense as-is and I'm going to merge it.

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Review by foolip — COMMENTED** [review]
Oops, I had a pending comment/suggestion.

**Comment by ddbeck** [comment]
@foolip Sorry for moving ahead prematurely on that. I opened #8040.

**Comment by foolip** [comment]
@ddbeck I hasn't submitted the review, so it was my mistake. Thanks for fixing nevertheless!

**Pull request merged**